editor/editor_file_system.cpp

```diff
@@ -1872,6 +1872,10 @@ void EditorFileSystem::_update_file_icon_path(EditorFileSystemDirectory::FileInf
 		}
 	}
 
+	if (icon_path.is_empty() && !file_info->type.is_empty()) {
+		icon_path = EditorNode::get_singleton()->get_class_icon(file_info->type)->get_path();
```

Comment:
This is wrongly assuming that the result of `EditorNode::get_singleton()->get_class_icon(file_info->type)` is valid and can de-referenced.

Reply:
Fixed by #97491.